nemo_skills/prompt/few_shot_examples/minif2f.py

```diff
@@ -0,0 +1,475 @@
+minif2f_deepseek_fewshot = [
```

Comment:
Let's keep only 5 few-shots, I don't think we need more

Reply:
Maybe for models that are not trained for lean4 specifically we would need all the fewshots? like for 405b?

Maybe I could just give 5 examples by default?
```
examples_map = {
    "minif2f_deepseek_fewshot": minif2f_deepseek_fewshot[:5],
}
```

Reply:
Let's test it and see if using >5 has any benefit